mnemosyne/hermes_config.py

```
@@ -0,0 +1,82 @@
+    config_path = os.path.join(hermes_home, "config.yaml") if hermes_home else ""
+    if not config_path or not os.path.exists(config_path):
+        return None
+    try:
+        import yaml
+    except ImportError:
+        return read_config_key_without_yaml(config_path, key)
+
+    try:
+        with open(config_path) as f:
+            config = yaml.safe_load(f) or {}
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)